scripts/accountancy/export-thirdpartyaccount.php

```diff
@@ -128,7 +128,7 @@ function launch_export()
 $sql .= " AND s.fk_pays = cp.rowid";
 if (! empty($date_start) && ! empty($date_end))
 	$sql .= " AND f.datec >= '" . $db->idate($date_start) . "' AND f.datec <= '" . $db->idate($date_end) . "'";
-$sql .= " AND f.entity = " . $conf->entity;
+$sql .= " AND f.entity IN (".getEntity('invoice').")";
```

Comment:
Here we are in module accountancy, and in this module invoices are never shared (an invoice can only be accounted into one and only one company). So i think we should have 
getEntity('invoice',0)
here.